Deployment on Minikube 1.38.1 fails despite following instructions

Hi there,

I have a test setup based on Minikube that uses the instructions documented at https://piraeus.io/docs/v2.10.0/how-to/minikube/ to deploy piraeus v2.10.5 on Minikube. This used to work for previous Minikube versions, but with the current v1.38.1 I am getting an error for the `drbd-module-loader` init container:

```
ERROR: modpost: "memcmp" [/tmp/pkg/drbd-9.2.14/drbd/build-current/drbd.ko] undefined!
ERROR: modpost: "__mutex_init" [/tmp/pkg/drbd-9.2.14/drbd/build-current/drbd.ko] undefined!
ERROR: modpost: "mutex_lock_interruptible" [/tmp/pkg/drbd-9.2.14/drbd/build-current/drbd.ko] undefined!
ERROR: modpost: "down_trylock" [/tmp/pkg/drbd-9.2.14/drbd/build-current/drbd.ko] undefined!
ERROR: modpost: "jiffies_to_msecs" [/tmp/pkg/drbd-9.2.14/drbd/build-current/drbd.ko] undefined!
ERROR: modpost: "errno_to_blk_status" [/tmp/pkg/drbd-9.2.14/drbd/build-current/drbd.ko] undefined!
ERROR: modpost: "_raw_spin_unlock_irqrestore" [/tmp/pkg/drbd-9.2.14/drbd/build-current/drbd.ko] undefined!
ERROR: modpost: "blk_set_stacking_limits" [/tmp/pkg/drbd-9.2.14/drbd/build-current/drbd.ko] undefined!
ERROR: modpost: "idr_destroy" [/tmp/pkg/drbd-9.2.14/drbd/build-current/drbd.ko] undefined!
ERROR: modpost: "bd_unlink_disk_holder" [/tmp/pkg/drbd-9.2.14/drbd/build-current/drbd.ko] undefined!
```

Complete log below...

Any ideas?

Thanks,
Thilo

```
$ k -n piraeus-datastore logs pod/linstor-satellite.minikube-5fdbw --container drbd-module-loader -f
+ [ ! -e /proc/drbd ]
+ apt-get update
Hit:1 http://archive.ubuntu.com/ubuntu noble InRelease
Get:2 http://security.ubuntu.com/ubuntu noble-security InRelease [126 kB]
Get:3 http://archive.ubuntu.com/ubuntu noble-updates InRelease [126 kB]
Get:4 http://archive.ubuntu.com/ubuntu noble-backports InRelease [126 kB]
Get:5 http://security.ubuntu.com/ubuntu noble-security/restricted amd64 Packages [3315 kB]
Get:6 http://archive.ubuntu.com/ubuntu noble-updates/universe amd64 Packages [2027 kB]
Get:7 http://archive.ubuntu.com/ubuntu noble-updates/main amd64 Packages [2285 kB]
Get:8 http://security.ubuntu.com/ubuntu noble-security/main amd64 Packages [1900 kB]
Get:9 http://archive.ubuntu.com/ubuntu noble-updates/restricted amd64 Packages [3486 kB]
Get:10 http://security.ubuntu.com/ubuntu noble-security/multiverse amd64 Packages [34.8 kB]
Get:11 http://security.ubuntu.com/ubuntu noble-security/universe amd64 Packages [1261 kB]
Get:12 http://archive.ubuntu.com/ubuntu noble-updates/multiverse amd64 Packages [38.1 kB]
Get:13 http://archive.ubuntu.com/ubuntu noble-backports/main amd64 Packages [49.5 kB]
Get:14 http://archive.ubuntu.com/ubuntu noble-backports/universe amd64 Packages [34.6 kB]
Fetched 14.8 MB in 1s (16.4 MB/s)
Reading package lists...
+ apt-get install -y flex bison bc libssl-dev libelf-dev
Reading package lists...
Building dependency tree...
Reading state information...
The following additional packages will be installed:
  libfl-dev libfl2 libssl3t64 libzstd-dev m4 openssl zlib1g-dev
Suggested packages:
  bison-doc build-essential flex-doc libssl-doc m4-doc
The following NEW packages will be installed:
  bc bison flex libelf-dev libfl-dev libfl2 libssl-dev libzstd-dev m4
  zlib1g-dev
The following packages will be upgraded:
  libssl3t64 openssl
2 upgraded, 10 newly installed, 0 to remove and 64 not upgraded.
Need to get 8073 kB of archives.
After this operation, 20.8 MB of additional disk space will be used.
Get:1 http://archive.ubuntu.com/ubuntu noble/main amd64 m4 amd64 1.4.19-4build1 [244 kB]
Get:2 http://archive.ubuntu.com/ubuntu noble/main amd64 flex amd64 2.6.4-8.2build1 [301 kB]
Get:3 http://archive.ubuntu.com/ubuntu noble-updates/main amd64 libssl3t64 amd64 3.0.13-0ubuntu3.7 [1942 kB]
Get:4 http://archive.ubuntu.com/ubuntu noble-updates/main amd64 openssl amd64 3.0.13-0ubuntu3.7 [1003 kB]
Get:5 http://archive.ubuntu.com/ubuntu noble/main amd64 bc amd64 1.07.1-3ubuntu4 [85.3 kB]
Get:6 http://archive.ubuntu.com/ubuntu noble/main amd64 bison amd64 2:3.8.2+dfsg-1build2 [749 kB]
Get:7 http://archive.ubuntu.com/ubuntu noble-updates/main amd64 zlib1g-dev amd64 1:1.3.dfsg-3.1ubuntu2.1 [894 kB]
Get:8 http://archive.ubuntu.com/ubuntu noble-updates/main amd64 libzstd-dev amd64 1.5.5+dfsg2-2build1.1 [364 kB]
Get:9 http://archive.ubuntu.com/ubuntu noble-updates/main amd64 libelf-dev amd64 0.190-1.1ubuntu0.1 [68.5 kB]
Get:10 http://archive.ubuntu.com/ubuntu noble/main amd64 libfl2 amd64 2.6.4-8.2build1 [9768 B]
Get:11 http://archive.ubuntu.com/ubuntu noble/main amd64 libfl-dev amd64 2.6.4-8.2build1 [6250 B]
Get:12 http://archive.ubuntu.com/ubuntu noble-updates/main amd64 libssl-dev amd64 3.0.13-0ubuntu3.7 [2407 kB]
debconf: delaying package configuration, since apt-utils is not installed
Fetched 8073 kB in 0s (16.2 MB/s)
Selecting previously unselected package m4.
(Reading database ... 10911 files and directories currently installed.)
Preparing to unpack .../m4_1.4.19-4build1_amd64.deb ...
Unpacking m4 (1.4.19-4build1) ...
Selecting previously unselected package flex.
Preparing to unpack .../flex_2.6.4-8.2build1_amd64.deb ...
Unpacking flex (2.6.4-8.2build1) ...
Preparing to unpack .../libssl3t64_3.0.13-0ubuntu3.7_amd64.deb ...
Unpacking libssl3t64:amd64 (3.0.13-0ubuntu3.7) over (3.0.13-0ubuntu3.5) ...
Setting up libssl3t64:amd64 (3.0.13-0ubuntu3.7) ...
(Reading database ... 11042 files and directories currently installed.)
Preparing to unpack .../0-openssl_3.0.13-0ubuntu3.7_amd64.deb ...
Unpacking openssl (3.0.13-0ubuntu3.7) over (3.0.13-0ubuntu3.5) ...
Selecting previously unselected package bc.
Preparing to unpack .../1-bc_1.07.1-3ubuntu4_amd64.deb ...
Unpacking bc (1.07.1-3ubuntu4) ...
Selecting previously unselected package bison.
Preparing to unpack .../2-bison_2%3a3.8.2+dfsg-1build2_amd64.deb ...
Unpacking bison (2:3.8.2+dfsg-1build2) ...
Selecting previously unselected package zlib1g-dev:amd64.
Preparing to unpack .../3-zlib1g-dev_1%3a1.3.dfsg-3.1ubuntu2.1_amd64.deb ...
Unpacking zlib1g-dev:amd64 (1:1.3.dfsg-3.1ubuntu2.1) ...
Selecting previously unselected package libzstd-dev:amd64.
Preparing to unpack .../4-libzstd-dev_1.5.5+dfsg2-2build1.1_amd64.deb ...
Unpacking libzstd-dev:amd64 (1.5.5+dfsg2-2build1.1) ...
Selecting previously unselected package libelf-dev:amd64.
Preparing to unpack .../5-libelf-dev_0.190-1.1ubuntu0.1_amd64.deb ...
Unpacking libelf-dev:amd64 (0.190-1.1ubuntu0.1) ...
Selecting previously unselected package libfl2:amd64.
Preparing to unpack .../6-libfl2_2.6.4-8.2build1_amd64.deb ...
Unpacking libfl2:amd64 (2.6.4-8.2build1) ...
Selecting previously unselected package libfl-dev:amd64.
Preparing to unpack .../7-libfl-dev_2.6.4-8.2build1_amd64.deb ...
Unpacking libfl-dev:amd64 (2.6.4-8.2build1) ...
Selecting previously unselected package libssl-dev:amd64.
Preparing to unpack .../8-libssl-dev_3.0.13-0ubuntu3.7_amd64.deb ...
Unpacking libssl-dev:amd64 (3.0.13-0ubuntu3.7) ...
Setting up libzstd-dev:amd64 (1.5.5+dfsg2-2build1.1) ...
Setting up bc (1.07.1-3ubuntu4) ...
Setting up m4 (1.4.19-4build1) ...
Setting up libssl-dev:amd64 (3.0.13-0ubuntu3.7) ...
Setting up libfl2:amd64 (2.6.4-8.2build1) ...
Setting up zlib1g-dev:amd64 (1:1.3.dfsg-3.1ubuntu2.1) ...
Setting up bison (2:3.8.2+dfsg-1build2) ...
update-alternatives: using /usr/bin/bison.yacc to provide /usr/bin/yacc (yacc) in auto mode
update-alternatives: warning: skip creation of /usr/share/man/man1/yacc.1.gz because associated file /usr/share/man/man1/bison.yacc.1.gz (of link group yacc) doesn't exist
Setting up openssl (3.0.13-0ubuntu3.7) ...
Setting up flex (2.6.4-8.2build1) ...
Setting up libfl-dev:amd64 (2.6.4-8.2build1) ...
Setting up libelf-dev:amd64 (0.190-1.1ubuntu0.1) ...
Processing triggers for libc-bin (2.39-0ubuntu8.5) ...
+ tar xzC /usr/src
+ uname -r
+ cut -d. -f1
+ uname -r
+ curl -fsSL https://www.kernel.org/pub/linux/kernel/v6.x/linux-6.6.95.tar.gz
+ uname -r
+ cd /usr/src/linux-6.6.95
+ zcat /proc/config.gz
+ make olddefconfig
#
# configuration written to .config
#
+ make modules_prepare
  SYNC    include/config/auto.conf.cmd
  CALL    scripts/checksyscalls.sh
  DESCEND objtool
  INSTALL libsubcmd_headers
  DESCEND bpf/resolve_btfids
  INSTALL libsubcmd_headers
+ cd /
+ uname -r
+ exec env LB_MAKEOPTS=KDIR=/usr/src/linux-6.6.95 LB_HOW=compile /entry.sh
Need a git checkout to regenerate drbd/.drbd_git_revision
make[1]: Entering directory '/tmp/pkg/drbd-9.2.14/drbd'

    Calling toplevel makefile of kernel source tree, which I believe is in
    KDIR=/usr/src/linux-6.6.95

make -C /usr/src/linux-6.6.95    "PRE_CFLAGS=" M=/tmp/pkg/drbd-9.2.14/drbd obj-m=dummy-for-compat.o dummy-for-compat-h.o
  COMPAT  have_sane_test_environment
  COMPAT  have_sane_test_environment_always_fail
  COMPAT  __vmalloc_has_2_params
  COMPAT  add_disk_returns_int
  COMPAT  before_4_13_kernel_read
  COMPAT  bio_alloc_has_4_params
  COMPAT  blk_alloc_disk_takes_queue_limits
  COMPAT  blkdev_get_by_path_has_holder_ops
  COMPAT  blkdev_issue_discard_takes_flags
  COMPAT  blkdev_issue_zeroout_discard
  COMPAT  blkdev_put_has_holder
  COMPAT  block_device_operations_open_takes_gendisk
  COMPAT  block_device_operations_release_takes_single_argument
  COMPAT  can_include_vermagic_h
  COMPAT  dax_direct_access_takes_mode
  COMPAT  fs_dax_get_by_bdev_takes_start_off
  COMPAT  fs_dax_get_by_bdev_takes_start_off_and_holder
  COMPAT  genl_policy_in_ops
  COMPAT  genlmsg_multicast_allns_has_4_params
  COMPAT  have_BIO_MAX_VECS
  COMPAT  have_CRYPTO_TFM_NEED_KEY
  COMPAT  have_GENHD_FL_NO_PART
  COMPAT  have_SHASH_DESC_ON_STACK
  COMPAT  have_WB_congested_enum
  COMPAT  have___bio_add_page
  COMPAT  have_allow_kernel_signal
  COMPAT  have_bdev_discard_granularity
  COMPAT  have_bdev_file_open_by_path
  COMPAT  have_bdev_max_discard_sectors
  COMPAT  have_bdev_nr_sectors
  COMPAT  have_bdev_open_by_path
  COMPAT  have_bdevname
  COMPAT  have_bdgrab
  COMPAT  have_bdi_congested
  COMPAT  have_bdi_congested_fn
  COMPAT  have_bio_advance_iter_single
  COMPAT  have_bio_alloc_clone
  COMPAT  have_bio_bi_bdev
  COMPAT  have_bio_bi_error
  COMPAT  have_bio_bi_opf
  COMPAT  have_bio_bi_status
  COMPAT  have_bio_clone_fast
  COMPAT  have_bio_max_segs
  COMPAT  have_bio_op_shift
  COMPAT  have_bio_set_dev
  COMPAT  have_bio_split_to_limits
  COMPAT  have_bio_start_io_acct
  COMPAT  have_bioset_init
  COMPAT  have_bioset_need_bvecs
  COMPAT  have_blk_alloc_disk
  COMPAT  have_blk_alloc_queue_rh
  COMPAT  have_blk_cleanup_disk
  COMPAT  have_blk_mode_t
  COMPAT  have_blk_opf_t
  COMPAT  have_blk_qc_t_make_request
  COMPAT  have_blk_qc_t_submit_bio
  COMPAT  have_blk_queue_flag_set
  COMPAT  have_blk_queue_make_request
  COMPAT  have_blk_queue_max_write_same_sectors
  COMPAT  have_blk_queue_merge_bvec
  COMPAT  have_blk_queue_split_bio
  COMPAT  have_blk_queue_split_q_bio
  COMPAT  have_blk_queue_split_q_bio_bioset
  COMPAT  have_blk_queue_update_readahead
  COMPAT  have_blk_queue_write_cache
  COMPAT  have_bvec_kmap_local
  COMPAT  have_counted_by
  COMPAT  have_d_inode
  COMPAT  have_debugfs_change_name
  COMPAT  have_disk_update_readahead
  COMPAT  have_enum_req_op
  COMPAT  have_fallthrough
  COMPAT  have_fs_dax_get_by_bdev
  COMPAT  have_generic_start_io_acct_q_rw_sect_part
  COMPAT  have_generic_start_io_acct_rw_sect_part
  COMPAT  have_genl_info_userhdr
  COMPAT  have_get_random_u32
  COMPAT  have_get_random_u32_below
  COMPAT  have_hd_struct
  COMPAT  have_ib_cq_init_attr
  COMPAT  have_ib_get_dma_mr
  COMPAT  have_idr_is_empty
  COMPAT  have_inode_lock
  COMPAT  have_kmap_local_page
  COMPAT  have_ktime_to_timespec64
  COMPAT  have_kvfree
  COMPAT  have_kvfree_rcu
  COMPAT  have_kvfree_rcu_mightsleep
  COMPAT  have_linux_unaligned_h
  COMPAT  have_list_is_first
  COMPAT  have_list_next_entry
  COMPAT  have_lookup_user_key
  COMPAT  have_max_send_recv_sge
  COMPAT  have_mempool_is_saturated
  COMPAT  have_nla_nest_start_noflag
  COMPAT  have_nla_parse_deprecated
  COMPAT  have_nla_put_64bit
  COMPAT  have_nla_strscpy
  COMPAT  have_part_stat_h
  COMPAT  have_part_stat_read_accum
  COMPAT  have_pointer_backing_dev_info
  COMPAT  have_proc_create_single
  COMPAT  have_queue_flag_discard
  COMPAT  have_queue_flag_stable_writes
  COMPAT  have_queue_limits_start_update
  COMPAT  have_rb_declare_callbacks_max
  COMPAT  have_refcount_inc
  COMPAT  have_req_noidle
  COMPAT  have_req_nounmap
  COMPAT  have_req_op_write
  COMPAT  have_req_op_write_zeroes
  COMPAT  have_req_write
  COMPAT  have_revalidate_disk_size
  COMPAT  have_sched_set_fifo
  COMPAT  have_sched_signal_h
  COMPAT  have_sendpage
  COMPAT  have_sendpage_ok
  COMPAT  have_set_capacity_and_notify
  COMPAT  have_shash_desc_zero
  COMPAT  have_simple_positive
  COMPAT  have_sk_use_task_frag
  COMPAT  have_sock_set_keepalive
  COMPAT  have_strscpy
  COMPAT  have_struct_bvec_iter
  COMPAT  have_struct_size
  COMPAT  have_submit_bio_noacct
  COMPAT  have_tasklet_setup
  COMPAT  have_tcp_sock_set_cork
  COMPAT  have_tcp_sock_set_keepcnt
  COMPAT  have_tcp_sock_set_keepidle
  COMPAT  have_tcp_sock_set_nodelay
  COMPAT  have_tcp_sock_set_quickack
  COMPAT  have_time64_to_tm
  COMPAT  have_timer_delete
  COMPAT  have_timer_setup
  COMPAT  have_timer_shutdown
  COMPAT  have_tls_get_record_type
  COMPAT  have_tls_tx_rx
  COMPAT  have_void_make_request
  COMPAT  have_void_submit_bio
  COMPAT  ib_alloc_pd_has_2_params
  COMPAT  ib_device_has_ops
  COMPAT  ib_post_send_const_params
  COMPAT  ib_query_device_has_3_params
  COMPAT  need_drbd_wrappers
  COMPAT  need_make_request_recursion
  COMPAT  need_recalc_sigpending
  COMPAT  part_stat_read_takes_block_device
  COMPAT  queue_limits_has_discard_zeroes_data
  COMPAT  queue_limits_has_features
  COMPAT  queue_limits_has_max_hw_discard_sectors
  COMPAT  queue_limits_has_max_hw_zone_append_sectors
  COMPAT  queue_limits_has_max_zone_append_sectors
  COMPAT  rdma_create_id_has_net_ns
  COMPAT  rdma_reject_has_reason_arg
  COMPAT  sk_data_ready_has_1_param
  COMPAT  sock_create_kern_has_netns_parameter
  COMPAT  sock_ops_returns_addr_len
  COMPAT  struct_gendisk_has_backing_dev_info
  UPD     /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/compat.h
  UPD     /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/.drbd_kernelrelease
  LN      build-current -> build-6.6.95/
  LN      compat.h -> build-6.6.95/compat.h
  LN      .compat_test -> build-6.6.95/.compat_test
  LN      compat.6.6.95.h -> build-6.6.95/compat.h
  LN      .compat_test.6.6.95 -> build-6.6.95/.compat_test
make -C /usr/src/linux-6.6.95    "PRE_CFLAGS=" M=/tmp/pkg/drbd-9.2.14/drbd obj-m=dummy-for-prep.o dummy-for-patch.o
make -C /tmp/pkg/drbd-9.2.14/drbd -f Makefile.spatch /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/compat.patch
  GENPATCHNAMES   6.6.95
  COCCISYN  70637d0c72ef81912210c230ffc5ef7d  6.6.95
  SPATCH  70637d0c72ef81912210c230ffc5ef7d  6.6.95
  You can create a new .tgz including this pre-computed compat patch
  by calling "echo drbd/drbd-kernel-compat/cocci_cache/70637d0c72ef81912210c230ffc5ef7d/compat.patch >>.filelist ; make tgz"
  CHK     build-6.6.95/compat.patch
  LN      /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/compat.patch -> ../drbd-kernel-compat/cocci_cache/70637d0c72ef81912210c230ffc5ef7d/compat.patch
set -- /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/compat.patch /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/compat.h;   \
test $1 -nt $2 || touch -r $1 $2
  SPLIT   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/compat.patch
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd-headers/compat.h
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd-headers/drbd_meta_data.h
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd-headers/drbd_protocol.h
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd-headers/drbd_strings.c
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd-headers/drbd_strings.h
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd-headers/drbd_transport.h
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd-headers/linux/drbd.h
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd-headers/linux/drbd_genl.h
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd-headers/linux/drbd_genl_api.h
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd-headers/linux/drbd_limits.h
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd-headers/linux/genl_magic_func-genl_register_family_with_ops_groups.h
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd-headers/linux/genl_magic_func-genl_register_mc_group.h
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd-headers/linux/genl_magic_func.h
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd-headers/linux/genl_magic_struct.h
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd-headers/windrbd/windrbd_ioctl.h
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd-kernel-compat/cocci_macros.h
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd-kernel-compat/drbd_wrappers.c
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd-kernel-compat/drbd_wrappers.h
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd-kernel-compat/handshake/genl.c
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd-kernel-compat/handshake/genl.h
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd-kernel-compat/handshake/handshake.h
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd-kernel-compat/handshake/netlink.c
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd-kernel-compat/handshake/request.c
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd-kernel-compat/handshake/tlshd.c
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd-kernel-compat/linux/libnvdimm.h
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd-kernel-compat/linux/lru_cache.h
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd-kernel-compat/linux/overflow.h
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd-kernel-compat/lru_cache.c
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd-kernel-compat/net/handshake.h
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd-kernel-compat/nsecs_to_jiffies.c
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd-kernel-compat/uapi/linux/handshake.h
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd-kernel-compat/uapi/linux/sched/types.h
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd_actlog.c
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd_bitmap.c
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd_dax_pmem.c
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd_dax_pmem.h
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd_debugfs.c
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd_debugfs.h
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd_int.h
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd_interval.c
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd_interval.h
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd_kref_debug.c
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd_kref_debug.h
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd_main.c
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd_nl.c
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd_nla.c
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd_nla.h
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd_polymorph_printk.h
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd_proc.c
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd_receiver.c
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd_req.c
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd_req.h
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd_sender.c
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd_state.c
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd_state.h
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd_state_change.h
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd_transport.c
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd_transport_lb-tcp.c
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd_transport_rdma.c
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd_transport_tcp.c
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd_transport_template.c
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd_vli.h
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/kref_debug.c
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/kref_debug.h
  PATCH   /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/linux/drbd_config.h
  COPY    /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/Kbuild
  GEN     /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd_buildtag.c 
  GEN     /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/.kernel.config.gz 
  COPY    /tmp/pkg/drbd-9.2.14/drbd/build-6.6.95/drbd-kernel-compat/handshake/Kbuild
test -e build-current && \
make -C /usr/src/linux-6.6.95    "PRE_CFLAGS=" M=/tmp/pkg/drbd-9.2.14/drbd/build-current modules
grep: ./Module.symvers: No such file or directory
  CC [M]  /tmp/pkg/drbd-9.2.14/drbd/build-current/drbd_buildtag.o
  CC [M]  /tmp/pkg/drbd-9.2.14/drbd/build-current/drbd_bitmap.o
  CC [M]  /tmp/pkg/drbd-9.2.14/drbd/build-current/drbd_proc.o
  CC [M]  /tmp/pkg/drbd-9.2.14/drbd/build-current/drbd_sender.o
  CC [M]  /tmp/pkg/drbd-9.2.14/drbd/build-current/drbd_receiver.o
  CC [M]  /tmp/pkg/drbd-9.2.14/drbd/build-current/drbd_req.o
  CC [M]  /tmp/pkg/drbd-9.2.14/drbd/build-current/drbd_actlog.o
  CC [M]  /tmp/pkg/drbd-9.2.14/drbd/build-current/drbd_main.o
  COPY    /tmp/pkg/drbd-9.2.14/drbd/build-current/drbd_strings.c
  CC [M]  /tmp/pkg/drbd-9.2.14/drbd/build-current/drbd_strings.o
  CC [M]  /tmp/pkg/drbd-9.2.14/drbd/build-current/drbd_nl.o
  CC [M]  /tmp/pkg/drbd-9.2.14/drbd/build-current/drbd_interval.o
  CC [M]  /tmp/pkg/drbd-9.2.14/drbd/build-current/drbd_state.o
  CC [M]  /tmp/pkg/drbd-9.2.14/drbd/build-current/drbd_nla.o
  CC [M]  /tmp/pkg/drbd-9.2.14/drbd/build-current/drbd_transport.o
  CC [M]  /tmp/pkg/drbd-9.2.14/drbd/build-current/drbd-kernel-compat/nsecs_to_jiffies.o
  CC [M]  /tmp/pkg/drbd-9.2.14/drbd/build-current/drbd-kernel-compat/lru_cache.o
  CC [M]  /tmp/pkg/drbd-9.2.14/drbd/build-current/drbd-kernel-compat/drbd_wrappers.o
  CC [M]  /tmp/pkg/drbd-9.2.14/drbd/build-current/drbd_debugfs.o
  LD [M]  /tmp/pkg/drbd-9.2.14/drbd/build-current/drbd.o
  CC [M]  /tmp/pkg/drbd-9.2.14/drbd/build-current/drbd_transport_tcp.o
  CC [M]  /tmp/pkg/drbd-9.2.14/drbd/build-current/drbd_transport_lb-tcp.o
grep: ./Module.symvers: No such file or directory
  MODPOST /tmp/pkg/drbd-9.2.14/drbd/build-current/Module.symvers
WARNING: Module.symvers is missing.
         Modules may not have dependencies or modversions.
         You may get many unresolved symbol errors.
         You can set KBUILD_MODPOST_WARN=1 to turn errors into warning
         if you want to proceed at your own risk.
ERROR: modpost: "memcmp" [/tmp/pkg/drbd-9.2.14/drbd/build-current/drbd.ko] undefined!
ERROR: modpost: "__mutex_init" [/tmp/pkg/drbd-9.2.14/drbd/build-current/drbd.ko] undefined!
ERROR: modpost: "mutex_lock_interruptible" [/tmp/pkg/drbd-9.2.14/drbd/build-current/drbd.ko] undefined!
ERROR: modpost: "down_trylock" [/tmp/pkg/drbd-9.2.14/drbd/build-current/drbd.ko] undefined!
ERROR: modpost: "jiffies_to_msecs" [/tmp/pkg/drbd-9.2.14/drbd/build-current/drbd.ko] undefined!
ERROR: modpost: "errno_to_blk_status" [/tmp/pkg/drbd-9.2.14/drbd/build-current/drbd.ko] undefined!
ERROR: modpost: "_raw_spin_unlock_irqrestore" [/tmp/pkg/drbd-9.2.14/drbd/build-current/drbd.ko] undefined!
ERROR: modpost: "blk_set_stacking_limits" [/tmp/pkg/drbd-9.2.14/drbd/build-current/drbd.ko] undefined!
ERROR: modpost: "idr_destroy" [/tmp/pkg/drbd-9.2.14/drbd/build-current/drbd.ko] undefined!
ERROR: modpost: "bd_unlink_disk_holder" [/tmp/pkg/drbd-9.2.14/drbd/build-current/drbd.ko] undefined!
WARNING: modpost: suppressed 389 unresolved symbol warnings because there were too many)
make[3]: *** [scripts/Makefile.modpost:145: /tmp/pkg/drbd-9.2.14/drbd/build-current/Module.symvers] Error 1
make[2]: *** [Makefile:1876: modpost] Error 2
make[1]: Leaving directory '/tmp/pkg/drbd-9.2.14/drbd'
make[1]: *** [Makefile:248: kbuild] Error 2
make: *** [Makefile:131: module] Error 2

Could not find the expexted *.ko, see stderr for more details
```

--- a/docs/how-to/minikube.md
+++ b/docs/how-to/minikube.md
@@ -45,6 +45,7 @@ spec:
             zcat /proc/config.gz > .config
             make olddefconfig
             make modules_prepare
+            ln -snf /usr/lib/modules/$(uname -r)/Module.symvers .
             cd /
           fi
           exec env "LB_MAKEOPTS=KDIR=/usr/src/linux-$(uname -r)" LB_HOW=compile /entry.sh

